include/translatable-object.php

```diff
@@ -340,17 +340,24 @@ public function where_clause( $lang ) {
 	 *
 	 * @since 3.4
 	 *
-	 * @param int $limit Max number of objects to return. `-1` to return all of them.
+	 * @param int   $limit  Max number of objects to return. `-1` to return all of them.
+	 * @param array $args {
+	 * 		The object args.
+	 *
+	 * 		@type string|string[] $object_types A translated object type or an array of translated object types.
+	 * }
 	 * @return int[] Array of object IDs.
 	 *
 	 * @phpstan-param -1|positive-int $limit
+	 * @phpstan-param array<array<string, string>> $args
 	 * @phpstan-return list<positive-int>
 	 */
-	public function get_objects_with_no_lang( $limit ) {
-		$language_ids = $this->model->get_languages_list();
+	public function get_objects_with_no_lang( $limit, $args = array() ) {
+		$languages = $this->model->get_languages_list();
 
-		foreach ( $language_ids as $i => $language ) {
-			$language_ids[ $i ] = $language->get_tax_prop( $this->get_tax_language(), 'term_taxonomy_id' );
+		$language_ids = array();
+		foreach ( $languages as  $language ) {
```

Comment:
If I can see it, PHPCS will 😅
```suggestion
		foreach ( $languages as $language ) {
```